lib/logstash/inputs/http.rb

```diff
@@ -42,6 +42,10 @@ class LogStash::Inputs::Http < LogStash::Inputs::Base
   # Set the truststore password
   config :keystore_password, :validate => :password
 
+  # here you can describe how to decode specific content-types
+  # by default the default codec will be used
+  config :additional_codecs, :validate => :hash, :default => { "application/json" => "json" }
```

Comment:
I like the feature and the idea, but one concern is that codecs can be configured, and this only lets you refer to codecs by name, not by configuration. For example, `line { charset => "ISO8859-1" }` can't be done with this implementation.

Thoughts?